### ISSUE TITLE
Number of training which the result is averaged over

### ISSUE THREAD

**Issue by san9569** [zhengyuf/STED-gaze#9]
Hi. I'm very interested in your work.

So I've trained your training code several times.
I could confirm that the variation of results exists somewhat.
Can I know the number of training done to get the results in your paper?

**Comment by zhengyuf** [comment]
Hi San, 

By variation of results, do you mean the variation of evaluation metrics? If so, how big is the variation? Also, which table are you trying to reproduce?

Cheers,
Yufeng

**Comment by san9569** [comment]
Hi, Thank you for your quick response.

I'm supposed to reproduce Table 2. For example, in the case of gaze redirection, the difference between the first and second trials is about 0.6. It the results are reasonble?

Thank you

**Comment by san9569** [comment]
Hi, Thank you for your quick response.

I was about to reproduce Table 2. For example, in the case of gaze redirection, the difference between the first and second trials is about 0.6. Is the results are reasonable?

Thank you

**Comment by zhengyuf** [comment]
Hi,

Table 2 in the main paper is only based on 1 training trial.
But if you go to table 1 in the supplementary material, you can see that the gaze redirection error is below 2.4 even when the loss weights are not optimal, and we didn't cherry-pick experiment trials. So I think a difference of 0.6 is a bit big (I assume that you are talking about our final method, not the ablation ones in the first three rows of table 2, main paper.)

Cheers,
Yufeng
